server/metrics/metrics.go

```diff
@@ -1514,6 +1517,16 @@ var (
 		EventName,
 	})
 
+	HealthCheckStatusCount = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I think it could just be a gauge with a value of 0 or 1 to indicate healthy/unhealthy

Reply:
You can't add labels to a gauge, and I thought it would be nice to filter on the name of the health check